pyproject.toml

```diff
@@ -19,7 +19,7 @@ priority = "supplemental"
 python = ">=3.8.1,<3.11"
 
 [tool.poetry.dependencies.rasa-pro]
-version = "3.9.1"
+version = "3.9.2"
```

Comment:
Can you update the REAME saying that this repo is up-to-date with `3.9.2`. And I guess we could use `3.9.3` as it was released today.